src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -1757,7 +1755,7 @@ ChildNodeList(MultiNode *multiNode)
  * When citus.enable_unique_job_ids is off then only the local counter is
  * included to get repeatable results.
  */
-static uint64
+uint64
 UniqueJobId(void)
```

Comment:
I wonder whether this function should now be in a separate file.

Reply:
I was thinking the same, but since the Job struct also is defined in `multi_physical_planner` I found no better place to put it. I can extract everything related to jobs :)